packages/react-components/src/organisms/NewsAndEventsCard.tsx

```diff
@@ -57,17 +57,27 @@ const placeholders: Record<NewsAndEventsType, JSX.Element> = {
   Training: trainingPlaceholderIcon,
 };
 
-type NewsAndEventsCardProps = NewsAndEventsResponse;
+type NewsAndEventsCardProps = NewsOrEventResponse & { href: string };
 
 const NewsAndEventsCard: React.FC<NewsAndEventsCardProps> = ({
   type,
   title,
   thumbnail,
+  text,
   link,
   linkText,
   shortText,
   created,
+  href,
 }) => {
+  const titleComponent = text ? (
```

Comment:
Seems this ternary could use a test? The test name would also help describe the requirement since this looks strange in code (checking for `text` truthiness but then not using it)